app/models/manageiq/providers/redhat/infra_manager/ovirt_services.rb

```diff
@@ -0,0 +1,3 @@
+module ManageIQ::Providers::Redhat::InfraManager::OvirtServices
+  require_nested :OvirtServices
```

Comment:
There is an issue with this line. We do not have OvirtServices in OvirtServices directory.
We should have here:

  require_nested :Builder